examples/App.cpp

```diff
@@ -24,6 +24,12 @@ void tracedSubroutine(const std::unique_ptr<opentracing::Span>& parentSpan)
 
 void tracedFunction()
 {
+    //{
+    //jaegertracing::SpanContext spanContextWithUserIDs { {111, 222}, 333, 0, 0, jaegertracing::SpanContext::StrMap() }; // TraceId and SpanID must be != 0
+    //auto span = opentracing::Tracer::Global()->StartSpan(
+    //    "tracedFunction1", {jaegertracing::SelfRef(&spanContextWithUserIDs)});
+    //}
```

Comment:
remove?

Reply:
I wanted to leave this commented code as an example of SelfRef using. Will remove in a next commit.

Reply:
Removed